mona_sdk/client.py

```diff
@@ -242,3 +252,86 @@ def _create_client_response(
             "sent": total - failed,
             "failure_reasons": failure_reasons,
         }
+
+    @Decorators.refresh_token_if_needed
+    def upload_config(self, config, commit_message):
+        """
+        Uploads a new configuration, as a json-serializable dict.
+        The configuration file enables you to define how the exported data should be
+        aggregated and analyzed in Mona, as well as your insight and alerting
+        preferences.
+        Find more information on creating your configuration at:
+        https://docs.monalabs.io/.
+        :param config: (dict)
+            json-serializable dict of the following format:
+            {
+            "YOUR_COMPANY_TENANT_ID": <your_new_configuration>
+            }
+        :param commit_message: (str)
+        :return: A dict holding the upload data:
+        {
+            "success": <was the upload successful>, (bool)
+            "new_config_id": <the new configuration ID> (str)
+        }
+        """
+        config_to_upload = {
+            "config": config,
+            "author": self._api_key,
+            "commitMessage": commit_message,
```

Comment:
weird to use camel case here and underscores for user id - please be consistent...